xml/art_sle_ha_pmremote.xml

```diff
@@ -0,0 +1,428 @@
+     <para>On the cluster node, virtual machines are launced by &pace;.</para>
+    </listitem>
+    <listitem>
+     <para>The cluster connects to the &pmremote; service of the virtual
+      machines.</para>
+    </listitem>
+    <listitem>
+     <para>The virtual machines are integrated into the cluster by &pmremote;.
+     </para>
+    </listitem>
+   </orderedlist>
+
+   <para>
+    Apart from the concept, it is important to know the several distinctive
+    roles a virtual machine can serve in &pace; clusters:
+    <!--
+    In &pace; clusters, a virtual machine can serve in several distinctive
+    roles:-->
+   </para>
+   <itemizedlist>
+    <listitem>
+     <para>
+      A virtual machine can run a full cluster stack. In this case, a cluster
+      node is not managed by the cluster.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      The cluster can manage a virtual machine as a resource without being
+      aware of the services running inside the virtual machine. In this
+      scenario, the virtual machine is opaque to the cluster.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      A virtual machine can act as a cluster resource and run the
+      &pmremote; service daemon.
+      This allows the cluster to manage services running on the virtual
+      machine. In this situation, the virtual machine is transparent to
+      the cluster.
+     </para>
+    </listitem>
+   </itemizedlist>
+  </sect1>
+
+ <sect1 xml:id="sec.ha.pmremote.install">
+  <title>Installation</title>
+  <para>Bascically, the installation consists of these steps:</para>
+  <orderedlist>
+   <listitem>
+    <para><xref linkend="sec.ha.pmremote.config.phys.host"/></para>
+   </listitem>
+   <listitem>
+    <para><xref linkend="sec.ha.pmremote.config.kvm.guest"/></para>
+   </listitem>
+   <listitem>
+    <para><xref linkend="sec.ha.pmremote.integrate.guestsintocluster"/></para>
+   </listitem>
+  </orderedlist>
+
+  <sect2 xml:id="sec.ha.pmremote.config.phys.host">
+   <title>Configuring the Physical Host</title>
+   <para>
+    On the physical host <systemitem class="domainname">&wsI;</systemitem>
+    do the following:
+   </para>
+   <procedure>
+    <!--<title>Configuring the Physical Host</title>-->
+    <step>
+     <para>Install your cluster as described in the <citetitle>&instquick;</citetitle>.
+      <!--<xref linkend="art.ha.install.quick"/>-->
+     </para>
+    </step>
+    <step>
+     <para>Make sure you open the TCP port 3121 in your firewall settings.</para>
+     <remark>toms 2017-03-23: what about the ports 2224 and 21064, and UDP port 5405?
+     Should they be opened as well?</remark>
+    </step>
+    <step>
+     <para>
+      Install the packages <package>acemaker-remote</package>
+      and <package>resource-agents</package>.</para>
+    </step>
+    <step>
+     <para>Configure the cluster nodes:</para>
+     <remark>toms 2017-03-23: Should we add all remote nodes into csync.cfg and /etc/hosts ?</remark>
```

Comment:
Good question. It  makes sense to add guest nodes into csync.cfg at least for the synchronization of the authkey, although the corosync configurations files listed in csync.cfg are basically useless for guest nodes. And it makes sen to add guest nodes into /etc/hosts.